<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-200</x>
      <y>0</y>
      <width>1901</width>
      <height>1131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>1121</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">;background-color:rgb(0, 211, 214);
border-radius:15px</string>
     </property>
     <widget class="QLabel" name="mshop">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins SemiBold</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 211, 214);

color:rgb(175, 0, 0)</string>
      </property>
      <property name="text">
       <string>MShop</string>
      </property>
     </widget>
     <widget class="QLabel" name="follow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Follow us</string>
      </property>
     </widget>
     <widget class="QLabel" name="instargram">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>D:/Instagram-icon-Social-media-emoticons-social-media-Software_482602_wh860-removebg-preview.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="search">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
padding-left:20px</string>
      </property>
      <property name="text">
       <string>Tìm kiếm sản phẩm</string>
      </property>
     </widget>
     <widget class="QLabel" name="beyblade">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins SemiBold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 211, 214);

color:rgb(175, 0, 0)</string>
      </property>
      <property name="text">
       <string>Beyblade</string>
      </property>
     </widget>
     <widget class="QLabel" name="krgavv">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>120</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins SemiBold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 211, 214);

color:rgb(175, 0, 0)</string>
      </property>
      <property name="text">
       <string>Kamen rider Gavv</string>
      </property>
     </widget>
     <widget class="QLabel" name="galaxyos">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>120</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins SemiBold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 211, 214);

color:rgb(175, 0, 0)</string>
      </property>
      <property name="text">
       <string>Galaxy Orbit Set</string>
      </property>
     </widget>
     <widget class="QLabel" name="zikudr">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins SemiBold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 211, 214);

color:rgb(175, 0, 0)</string>
      </property>
      <property name="text">
       <string>Ziku Driver</string>
      </property>
     </widget>
     <widget class="QLabel" name="facebook">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Facebook_icon_2013.svg.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="englishorvietnamese">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Tiếng Việt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>nhatnamnguyen</string>
      </property>
     </widget>
     <widget class="QLabel" name="searchicon">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>D:/pngtree-vector-search-icon-png-image_320926-removebg-preview.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="cart">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>70</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/pngtree-shopping-cart-convenient-icon-image_1287807-removebg-preview.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>991</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>361</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>749</width>
          <height>359</height>
         </rect>
        </property>
        <widget class="QLabel" name="bxcx">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>221</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>KR Gavv</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>361</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>749</width>
          <height>359</height>
         </rect>
        </property>
        <widget class="QLabel" name="boost">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>281</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BeyBlade Burst</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_11">
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>361</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>749</width>
          <height>359</height>
         </rect>
        </property>
        <widget class="QLabel" name="dranbuss">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BeyBlade X</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QScrollArea" name="scrollArea_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>971</width>
         <height>511</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>969</width>
          <height>509</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
